iocAdmin/src/iocReleaseCreateDb.py

```diff
@@ -47,19 +47,25 @@ def export_db_file( module_versions, path=None):
         x = key.replace("_MODULE_VERSION","",1)
         if idx >= idxMax:
             break
-        print('record(stringin, "$(IOC):RELEASE%02d") {' % idx, file=out_file)
+        print('record(%s, "$(IOC):RELEASE%02d") {' % ('lsi' if is_epics7 else 'stringin', idx), file=out_file)
+        if is_epics7:
+            print('  field(INP, {const:"%s"})' % module_version, file=out_file)
+        else:
+            print('  field(VAL, "%s")' % (module_version[:38] if not is_epics7 else module_version), file=out_file) # Truncate module_version at 40 chars (including NULL terminator). Otherwise this DB will fail to load.
         print('  field(DESC, "%s")' % x, file=out_file)
         print('  field(PINI, "YES")', file=out_file) 
-        print('  field(VAL, "%s")' % module_version[:38], file=out_file) # Truncate module_version at 40 chars (including NULL terminator). Otherwise this DB will fail to load.
         print('  #field(ASG, "some read only grp")', file=out_file) 
         print('}', file=out_file)
         idx = idx + 1
 
     while idx < idxMax:
-        print('record(stringin, "$(IOC):RELEASE%02d") {' % idx, file=out_file)
+        print('record(%s, "$(IOC):RELEASE%02d") {' % ('lsi' if is_epics7 else 'stringin', idx), file=out_file)
         print('  field(DESC, "Not Applicable")', file=out_file)
-        print('  field(PINI, "YES")', file=out_file) 
-        print('  field(VAL, "Not Applicable")', file=out_file)
+        print('  field(PINI, "YES")', file=out_file)
+        if is_epics7:
+            print('  field(INP, {const:"Not Applicable"})', file=out_file)
+        else:
+            print('  field(VAL, "Not Applicable")', file=out_file)
```

Comment:
Why use `{const:` for INP and not for VAL?

Reply:
`VAL` doesn't work for lsi I guess? when I used it, the db would fail to load with:
```
Can't set "test.VAL" to "test"  : Error (512,11)
ERROR:  at or before ')' in path "."  file "test.db" line 2

 2 |    field(VAL, "test")

ERROR failed to load 'test.db'
ERROR: Failed to load: test.db
```